recipes/r-vision/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "vision" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "52bb4aa13343e423e7fe51f5608fb9f4698afacfbefe968f3bf9fa6e75ede2b2" %}
+
+package:
+  name: r-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/YosefLab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - r-base
+    - r-fastica
+    - r-igraph
+    - r-irlba
+    - r-jsonlite
+    - r-loe
+    - r-logging
+    - parallel
+    - r-pbmcapply
+    - r-matrix
+    - r-matrixstats
+    - r-mclust
+    - r-mime
+    - r-plumber
+    - r-rann
+    - r-rcpp
+    - r-rsvd
+    - r-rtsne
+    - r-vegan
+    - r-wordspace
+  run:
+    - r-base
+    - r-fastica
+    - r-igraph
+    - r-irlba
+    - r-jsonlite
+    - r-loe
+    - r-logging
+    - parallel
+    - r-pbmcapply
+    - r-matrix
+    - r-matrixstats
+    - r-mclust
+    - r-mime
+    - r-plumber
+    - r-rann
+    - r-rcpp
+    - r-rsvd
+    - r-rtsne
+    - r-vegan
+    - r-wordspace
+
+test:
+  commands:
+    - $R -e "library('VISION')"
+
+about:
+  home: https://github.com/YosefLab/VISION
+  license: MIT
```

Comment:
Can you add a `license_file: LICENSE` line so a copy of the license is included?